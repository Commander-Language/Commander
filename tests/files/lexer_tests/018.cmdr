const _abc123_ = "_abc123_";
