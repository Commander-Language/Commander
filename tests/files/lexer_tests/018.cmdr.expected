CONST 'const'
VARIABLE '_abc123_'
EQUALS '='
STRINGVAL
[
STRINGLITERAL '_abc123_'
]
SEMICOLON ';'
END_OF_FILE
